### DISCUSSION TITLE
When using Imagemagick to compile svg in the buster

### DISCUSSION THREAD

**Discussion by 330595524** [ImageMagick/ImageMagick#5698] [Development]
### ImageMagick version

7.1.0-25

### Operating system

Linux

### Operating system, version and so on

buster

### Description

When using Imagemagick to compile svg in the buster, the error ERROR occurs when using the WriteImage method_ WAND: wand contains no images `MagickWand-2' @ error/magick-image.c/MagickWriteImage/13284

### Steps to Reproduce

import (
	"gopkg.in/gographics/imagick.v3/imagick"
)

image := imagick.NewMagickWand()
err := image.WriteImage("xxx.png"))

err.....

**Comment by snibgo** [comment]
This is an ImageMagick forum. Your question is about IMagick, which is a different product.

But it looks as if you are trying to write an image to a file without first reading or creating an image.
